### ISSUE TITLE
On how to store generated resources

### ISSUE THREAD

**Issue by josebreijo** [wizeline/FE.X#9]
I know we're starting yet, but looking forward we should decide a better way to keep track of all the job done _(talks, lightning, committees and their deliveries)_.

Those generated resources should be tracked in another repository referenced from this one. My interest in this is that, once [this list](https://github.com/wizeline/FE.X/tree/master/talks/full-fledged#talks-weve-done) grows bigger it will be hard to keep track on.

Each generated resource needs to be quickly browsable, and contain all the docs, demos, etc. That way it's easier to reuse by us in the future and easy to use by the 🥇**FE.X** 💯 audience.

This repository should be used as a reference on what we're doing and how to be part of it (how to contribute, requesting talks, discussions, etc.).

**Comment by memorammstein** [comment]
Agree. The list was always meant to be only a reference of what we've done, I don't think there has ever been no real intention of storing the actual talks' resources in the repo.
